packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -543,29 +539,18 @@ function commitLifeCycles(
       return;
     }
     case Profiler: {
-      if (enableProfilerTimer) {
+      if (enableProfiling) {
         const onRender = finishedWork.memoizedProps.onRender;
 
-        if (enableSchedulerTracing) {
-          onRender(
-            finishedWork.memoizedProps.id,
-            current === null ? 'mount' : 'update',
-            finishedWork.actualDuration,
-            finishedWork.treeBaseDuration,
-            finishedWork.actualStartTime,
-            getCommitTime(),
-            finishedRoot.memoizedInteractions,
-          );
-        } else {
-          onRender(
-            finishedWork.memoizedProps.id,
-            current === null ? 'mount' : 'update',
-            finishedWork.actualDuration,
-            finishedWork.treeBaseDuration,
-            finishedWork.actualStartTime,
-            getCommitTime(),
-          );
-        }
```

Comment:
We never actually used this combination anywhere (in www, fbsource, or open source).